<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBoxDialog</class>
 <widget class="QDialog" name="AddBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить ящик</string>
  </property>
  <property name="windowIcon">
   <iconset resource="diagn.qrc">
    <normaloff>:/Icons/images/empty-box.png</normaloff>:/Icons/images/empty-box.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Текст на ящике:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Куда добавить:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="beforeRB">
          <property name="text">
           <string>перед текущим элементом</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="afterRB">
          <property name="text">
           <string>после текущего элемента</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>beforeRB</tabstop>
  <tabstop>afterRB</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources>
  <include location="diagn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>AddBoxDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddBoxDialog</receiver>
   <slot>slotNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotNameChanged(QString)</slot>
 </slots>
</ui>
